src/prog_models/prognostics_model.py

```diff
@@ -939,9 +948,17 @@ def next_time(t, x):
             if (t >= save_pts[save_pt_index]):
                 save_pt_index += 1
                 update_all()
+            if config['progress']:
+                percentages = [1-val for val in event_state(x).values()]
+                percentages.append((t/horizon))
+                converted_iteration = min(100, max(percentages)*100)
```

Comment:
```suggestion
                converted_iteration = int(max(min(100, max(percentages)*100), 0))
```
To make sure converted_iteration is never less than 0. Also, we should probably use integer so it'll show 100%